### PR TITLE
docs(combine-to-osv): add basic service documentation

### PR DESCRIPTION
**Pull request by andrewpollock**
Primarily to better surface how to override an OSV record, originally described in #1604

### CODE REVIEW

**Review by cuixq — APPROVED** [review]
👍

File: vulnfeeds/cmd/combine-to-osv/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+* Reads from [`gs://cve-osv-conversion/parts`](https://storage.googleapis.com/cve-osv-conversion/index.html?prefix=parts/)
+* Merges with CVE data from NVD (obtained from GCS mirror maintained by [`download-cves`](../download-cves/mirror_nvd.sh))
+* Writes an OSV record to [`gs://cve-osv-conversion/osv-output`](https://storage.googleapis.com/cve-osv-conversion/index.html?prefix=osv-output/)
+  * This is the import source for [`cve-osv`](https://github.com/google/osv.dev/blob/2c22e9534a521c6c6350275427f80e481065ca39/source.yaml#L96)
+  * What gets written can be overridden by OSV records in [`gs://cve-osv-conversion/osv-output-overrides`](https://storage.googleapis.com/cve-osv-conversion/index.html?prefix=osv-output-overrides/)
+
+## Operational matters
+
+* Runs every 30 minutes as a [Kubernetes CronJob](https://github.com/google/osv.dev/blob/master/deployment/clouddeploy/gke-workers/base/combine-to-osv.yaml)
```
Comment (hogo6002) [inline vulnfeeds/cmd/combine-to-osv/README.md:27]: We are still running this every hour right? (“At minute 30 past every hour.”)

**Review by hogo6002 — APPROVED** [review]
Thanks Andrew!

File: vulnfeeds/cmd/combine-to-osv/README.md (hunk 1 above)
Comment (andrewpollock) [reply vulnfeeds/cmd/combine-to-osv/README.md:27]: Thanks, you're right. I was too hasty writing that bit. Fixed in b4f6eb33

File: vulnfeeds/cmd/combine-to-osv/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Possible edits to consider making:
+
+* remove or correct an incorrect `affected` entry
+* add a `withdrawn` field
+
+#### Procedure
+
+1. `gsutil cp gs://${OUTPUT_BUCKET}/osv-output/CVE-YYYY-NNNN.json`
+2. manually edit the file
+3. `gsutil cp gs://${INPUT_BUCKET/osv-output-overrides/CVE-YYYY-NNNN.json`
```
Comment (another-rex) [inline vulnfeeds/cmd/combine-to-osv/README.md:44]:
```suggestion
3. `gsutil cp gs://${INPUT_BUCKET}/osv-output-overrides/CVE-YYYY-NNNN.json`
```

**Review by another-rex — APPROVED** [review]

**Pull request merged**